### PR TITLE
Add some lorem and some ipsum

### PR DESCRIPTION
**Pull request by jbrimeyer-leepfrog**
I added some lorem ipsum so we have something to mess with

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -1 +1,3 @@
 This is the beginning
+
```
Comment (jbrimeyer-leepfrog) [inline readme.txt:2]: I love this!

Comment (jbrimeyer) [inline readme.txt:2]: Why do we need this?

Comment (jbrimeyer-leepfrog) [reply readme.txt:2]: Because I said so!

**Review by jbrimeyer — APPROVED** [review]
Looks good!

**Pull request merged**